### DISCUSSION TITLE
Include image file with "Export layers to images"?

### DISCUSSION THREAD

**Discussion by bryanseger** [sn4k3/UVtools#532] [Ideas]
I have been successfully using UVtools to tweak print files for the Stratasys Origin One printer using the "export layers to images" command, and it works great. The standard Origin One slicer includes a thumbnail image of the whole part, which shows up on the printer when you select the file, but I do not get that when using UVtools' "export layers to images" command. Is it possible to add a checkbox option to include a thumbnail image when using this command? I think a normal .jpeg would be all that's needed.

Also, within this command, is it possible to add the option to save into a zipped folder instead of a normal folder?

**Comment by sn4k3** [comment]
If you want the thumbnails included you can use File -> Export. Also you can save the thumbnails by click on save button near thumbnail.
If you want layers into zip you can use the File -> Convert -> Generic Zip

For more custom operation you must create a script with UVtools or with cmd.

Which file format does Stratasys Origin One uses?

**Reply by bryanseger** [reply]
Ok good deal. Thanks for the quick response. I'm still on version 3.2.0 (I know, I know...); I do not see "Generic Zip", but I did try both "Chitubox Zip" and the "Generic / Phrozen Zip" (I'm guessing this is the one you mentioned). Both of these compile the PNG images along with thumbnails, which is what I need, but also a gcode file (Chitu) or a manifest.uvtools file (Generic). I wonder if either of these will cause issues with the printer, but I can find out next week at work. Either way, I think I can make one of these options work.

Per Stratasys, we are to use Netfabb to slice the file, which simply exports the slices into a zipped folder of PNGs and a thumbnail. We then upload that zipped folder to Origin's online portal, where we have to use the online printer settings to actually control the printer. It's extremely locked down and not nearly as user friendly as Lychee, Chitubox, and especially UVtools. Netfabb just sucks to use, and I cannot turn off or control AA settings, so I figured out a way to use Lychee as my slicer, export as a .CWS file, open that into UVtools, and then export as a folder of PNGs. I still have to use Stratasys's online portal to control printer settings and to 'talk' to the printer, but at least I have a workaround for Netfabb.

Perhaps you know of a better workaround than mine?

**Reply by sn4k3** [reply]
> also a gcode file (Chitu) or a manifest.uvtools file (Generic). I wonder if either of these will cause issues with the printer

Printer will only use what it needs, anything else on the zip will not be considered.

> Per Stratasys, we are to use Netfabb to slice the file, which simply exports the slices into a zipped folder of PNGs and a thumbnail

>  so I figured out a way to use Lychee as my slicer, export as a .CWS file

> Perhaps you know of a better workaround than mine?

If your printer accept a zip of PNGs you can direct open .cws on UVtools and upload to printer as renamed .zip? 


> I'm still on version 3.2.0

I would upgrade ;)

**Reply by bryanseger** [reply]
Yes, I believe I can open the .CWS in UVtools, export as zip, then upload to the printer's portal. I believe that'll work but I can't find out until I get back to work next week.

I've updated on my home PC, but I have not yet on my work PC.
